test/test-server/backend.go

```diff
@@ -15,7 +15,7 @@ var (
 	TestHubUrl                 = url.URL{Scheme: "http", Host: "127.0.0.1:4096"}
 	TestKafkaRestProxyUrl      = url.URL{Scheme: "http", Host: "127.0.0.1:1025"}
 	TestFlinkGatewayUrl        = url.URL{Scheme: "http", Host: "127.0.0.1:1026"}
-	TestFlinkGatewayUrlPrivate = url.URL{Scheme: "http", Host: "127.0.0.1:1028"}
+	TestFlinkGatewayUrlPrivate = url.URL{Scheme: "http", Host: "127.0.0.1:1040"}
```

Comment:
Did this fix your error? Were we able to figure out why we couldn't use 1028? Maybe some other test uses it?

Reply:
No. This didn't fix my problem. I dropped the `spec.compute_pool_id` in the test